pkgs/top-level/all-packages.nix

```diff
@@ -13096,6 +13096,8 @@ with pkgs;
 
   ttf_bitstream_vera = callPackage ../data/fonts/ttf-bitstream-vera { };
 
+  ttf_envy_code_r = callPackage ../data/fonts/ttf-envy-code-r {};
```

Comment:
Any particular reason for underscores?  Dashes are preferred nowadays:  http://nixos.org/nixpkgs/manual/#sec-package-naming